packages/shift-next-routes/src/express/account-handler.js

```diff
@@ -36,19 +37,13 @@ module.exports = {
 
       if (response.status === 201) {
         extractCustomerId(req, response.data.data)
+        req.log && req.log.info(`Login by user ${req.session.customerId}`)
```

Comment:
This data feels more like `req.log.debug`, probably isn't something we care to see for every successful login, just my thoughts though!

Reply:
My argument for this being above debug is that it's a priviledge escalation - I'd normally record all instances of (attempted or successful) priviledge escalation for a service just for auditing purposes.